extension/src/shared/view/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export const generateID = () => {
```

Comment:
We already have time function in [here](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/0177aafd8102a2150d14e467cb2c9b405edf7593/extension/src/editor/containers/utils.ts#L64-L66). So could you define this function in `/shared/utils/id.ts` and change some code to use it?

Reply:
Updated!